src/dynamicprompts/commands/variable_commands.py

```diff
@@ -10,6 +10,7 @@ class VariableAssignmentCommand(Command):
     name: str
     value: Command
     immediate: bool
+    preserve: bool = False
```

Comment:
I wonder if there'd be a better name for this... `no_overwrite`, if that's what the end result is? 🤔

Reply:
I'm still new to Python, but in my general experience it is best to keep boolean value names positive. I can invert the default value and change this to `overwrite`

Reply:
Sounds good!